mne/report/report.py

```diff
@@ -101,55 +245,47 @@ def _fig_to_img(fig, image_format='png', scale=None, auto_close=True,
         fig = _ndarray_to_fig(img)
 
     output = BytesIO()
-    if scale is not None:
-        _scale_mpl_figure(fig, scale)
```

Comment:
There probably needs to be a deprecation cycle for this

Reply:
I've added the scaling back where it makes sense: `add_figs_to_section` and `add_slider_to_section`. I'm silently ignoring it for `add_images_to_section`, where it would previously not do any "actual" resizing, but simply scale the image via CSS – which would not make sense with our new, adaptive layout.